### ISSUE TITLE
Velero 1.14.0 tries to backup EmptyDir using PVC snapshots

### ISSUE THREAD

**Issue by fgleixner** [vmware-tanzu/velero#7986]
**What steps did you take and what happened:**

Upgraded Velero to 1.14.0. I used `--features=EnableCSI`  and `--default-snapshot-move-data` during installation.

I get many errors in backups:
```
            name: /kafka-controller-1 message: /Error backing up item error: /volume folio-prod-01/kafka-config has no PVC associated with it
             name: /kafka-controller-1 message: /Error backing up item error: /volume folio-prod-01/tmp has no PVC associated with it
             name: /kafka-controller-1 message: /Error backing up item error: /volume folio-prod-01/logs has no PVC associated with it
```
These are emptyDir and configmap Volumes. Velero 1.13.2 did not try to create CSI snapshots from these resources.

I also tries to add a resourcepolicy
```
version: v1
volumePolicies:
- conditions:
    volumeTypes:
      - emptydir
      - downwardAPI
      - configmap
      - cinder
  action:
    type: skip
```

but this did not help.

**What did you expect to happen:**

Ignore these resorce volumes by default as in 1.13.2

**Vote on this issue!**

This is an invitation to the Velero community to vote on issues, you can see the project's [top voted issues listed here](https://github.com/vmware-tanzu/velero/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).  
Use the "reaction smiley face" up to the right of this comment to vote.

- :+1: for "I would like to see this bug fixed as soon as possible"
- :-1: for "There are more important bugs to focus on right now"

**Comment by Lyndon-Li** [comment]
This looks like a volume policy problem introduced in 1.14. 

@shubham-pampattiwar Please help to check this problem. Looks like `ShouldPerformFSBackup` should take care of the volumes without PVC.

**Comment by shubham-pampattiwar** [comment]
@fgleixner @Lyndon-Li Duplicate of https://github.com/vmware-tanzu/velero/issues/7929
Closing this issue as fix PR is merged for 1.14.1
